source/sink-connector/fundamentals/write-strategies.txt

```diff
@@ -35,11 +35,40 @@ To learn how to modify the sink records your connector receives before your
 connector writes them to MongoDB, read the guide on
 :ref:`Sink Connector Post Processors <sink-fundamentals-post-processors>`.
 
-To learn more about bulk writes, see the MongoDB Java driver guide on
-`Bulk Writes <{+connector_driver_url_base+}fundamentals/crud/write-operations/bulk/>`__.
-
 To see a write model strategy implementation, see the source code of the
-:github:`InsertOneDefaultStrategy class <mongodb/mongo-kafka/blob/master/src/main/java/com/mongodb/kafka/connect/sink/writemodel/strategy/InsertOneDefaultStrategy.java>`.
+:github:`InsertOneDefaultStrategy class
+<mongodb/mongo-kafka/blob/master/src/main/java/com/mongodb/kafka/connect/sink/writemodel/strategy/InsertOneDefaultStrategy.java>`.
+
+.. _sink-connector-bulk-write-ops:
+
+Bulk Write Operations
+---------------------
+
+The sink connector writes data to MongoDB using bulk write operations.
+Bulk writes group multiple write operations, such as inserts,
+updates, or deletes, together. 
+
+By default, the sink connector performs ordered bulk writes, which
+guarantees the order of data changes. You can specify the
```

Comment:
Per our discussion, ordered bulk writes are the default and I believe the driver uses this because it would rather assume you want a write in the order you provided the models.